ludwig/trainers/trainer.py

```diff
@@ -1317,6 +1325,9 @@ def increase_batch_size(
         increase_batch_size_eval_split: str = TRAINING,
     ):
         """Uses the progress tracker to determine if the batch size should be increased."""
+        if increase_batch_size_on_plateau_max is None:
+            increase_batch_size_on_plateau_max = float("inf")  # If None, no hard limit on batch size.
```

Comment:
I feel weird about assigning this to a float when its typed as an int, though it works fine maybe we could use some really big integer constant instead?  Not super important

Reply:
This makes sense. To address both of the above comments, I think it could make sense to have some constant `MAX_POSSIBLE_BATCH_SIZE` that we use in both locations. I'll add that in for now– let me know what you think in the meantime. Thanks!